frontend/src/__tests__/cypress/cypress/e2e/applications/explore.cy.ts

```diff
@@ -21,4 +21,8 @@ describe('Explore Page', { testIsolation: false }, () => {
   it('card title should be visible', () => {
     cy.get('span').should('contain', 'Jupyter');
   });
+
+  it('card title should not be visible', () => {
+    cy.get('span').should('not.contain', 'RHOAI');
```

Comment:
Can you check for better selector than `span`. Additionally, using `not.contain` might not be effective if the text includes something like 'RHOAI abc'. Could you modify it to specifically check for the exact text 'RHOAI'?